counsel.el

```diff
@@ -2814,7 +2820,8 @@ A is the left hand side, B the right hand side."
 
 (ivy-set-actions
  'counsel-yank-pop
- '(("d" counsel-yank-pop-action-remove "delete")))
+ '(("d" counsel-yank-pop-action-remove "delete")
+   ("p" counsel-yank-pop-action-position "position")))
```

Comment:
Ditto re: saying "rotate" instead of "position". It would then follow that a more intuitive key should be "r" instead of "p".

Reply:
Rotate makes sense.